### ISSUE TITLE
Make README more quickstart-friendly

### ISSUE THREAD

**Issue by ryan-williams** [graphite-project/carbon#347]
* afaict the preferred command for running a carbon server is `bin/carbon-cache.py start`, but this is not documented in the README anywhere.
* the README makes assertions about paths under `$GRAPHITE_ROOT` but offers no explanation as to how someone should set `$GRAPHITE_ROOT`
    * some info on the readthedocs.org and/or wikidot imply that this should always be `/opt/graphite`, yet the assumption seems to be that a full clone of this repository will live there
    * is the assumption that this repository will be cloned to `/opt/graphite`?
* the closest thing to a link to more info on the README is its opening link to the [graphite-project](https://github.com/graphite-project) page; the only relevant-seeming thing there is the [github pages](https://github.com/graphite-project/graphite-project.github.io) repo, which points to [http://graphite-project.github.io/](http://graphite-project.github.io/), which serves up an nginx 404; no information anywhere indicates whether this is intentional / known / scheduled to be fixed or whether that github pages repo should be considered defunct.. 
* if the readthedocs page is or will soon be the canonical source for graphite documentation, can we link to it more clearly? graphite-web does but carbon and whisper do not.

**Comment by obfuscurity** [comment]
The README is not the official project documentation, nor is it intended to be. We are in the process of designing and launching a new project website that will address a lot of the problems with the existing documentation and website.

TL;DR - We're working on this and while I really do appreciate your feedback, these are known issues so I'd prefer to just close this issue out in lieu of actual pull requests.

**Comment by ryan-williams** [comment]
OK. Thoughts on at least clearly calling out the readthedocs page as being the canonical source? Having a little bit of info on the repo page and no link to the real documentation seems suboptimal. See previous point about graphite-web doing this but carbon and whisper not.

**Comment by obfuscurity** [comment]
Once the new website is up, that will be the canonical destination for Graphite users/visitors. It will have the necessary links to the documentation, or built-in content where applicable.

**Comment by ryan-williams** [comment]
is "the new website" the (seemingly WIP) one at http://graphite.readthedocs.org/en/latest/ (which is built from https://github.com/graphite-project/graphite-web/tree/master/docs, afaict), or will it live at http://graphite-project.github.io/? If the latter, is there any code written for it, or is the plan to port the readthedocs site to that new URL?

Sorry for all of the annoying questions, I'm just trying to find my way around and the answers to them do not seem to be readily available.

**Comment by obfuscurity** [comment]
It will exist on GitHub pages as http://graphiteapp.org/.
